countries/src/main/java/com/lambdaschool/countries/CountriesApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.lambdaschool.countries;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CountriesApplication {
+
+    static countryList ourCountries;
+    public static void main(String[] args)
+    {
+        ourCountries = new countryList();
```

Comment:
It's always a good idea to initialize everything before running your app. Great job!